examples/papermill_plugin/papermill_plugin/simple.py

```diff
@@ -32,6 +32,7 @@
     name="simple-nb",
     notebook_path=str(pathlib.Path(__file__).parent.absolute() / "nb_simple.ipynb"),
     render_deck=True,
+    disable_deck=False,
```

Comment:
Sorry, I think this option was deprecated recently. Use `enable_deck=True` please.
cc @samhita-alla

Reply:
sure

Reply:
i have fixed it now